app/navigation/GuarantorNavigator.scala

```diff
@@ -53,15 +51,25 @@ class GuarantorNavigator @Inject() extends BaseNavigator {
     case GuarantorVatPage => (userAnswers: UserAnswers) =>
       controllers.sections.guarantor.routes.GuarantorAddressController.onPageLoad(userAnswers.ern, userAnswers.lrn, NormalMode)
 
+    case GuarantorAddressPage => (userAnswers: UserAnswers) =>
+      controllers.sections.guarantor.routes.GuarantorCheckAnswersController.onPageLoad(userAnswers.ern, userAnswers.lrn)
+
     case _ =>
       (userAnswers: UserAnswers) => routes.IndexController.onPageLoad(userAnswers.ern)
   }
 
+  private val checkRoutes: Page => UserAnswers => Call = {
+
+    case _ =>
+      (userAnswers: UserAnswers) =>
+        controllers.sections.guarantor.routes.GuarantorCheckAnswersController.onPageLoad(userAnswers.ern, userAnswers.lrn)
```

Comment:
[Question] If the user answers `no` to `Is a guarantor required for this movement?` and go to check answers then click change and change answer to `yes` it will go back to check your answers. Do we need to handle this case with this ticket or is there a separate ticket. All changes just go back to check answers.

Reply:
It goes back to check your answers but you would then see the arranger as not provided so does allow you to go and then set that. I get what your saying but I don't believe we cater for it that way, not saying it couldn't be but it does work sufficiently as is.